ci/infra/testrunner/utils/utils.py

```diff
@@ -33,6 +35,36 @@ class Utils:
     def __init__(self, conf):
         self.conf = conf
 
+    @staticmethod
+    def chmod_recursive(directory, permissions):
+        os.chmod(directory, permissions)
+
+        for file in glob.glob(os.path.join(directory, "**/*"), recursive=True):
+            try:
+                os.chmod(file, permissions)
+            except Exception as ex:
+                logger.exception(ex)
+
+    @staticmethod
+    def cleanup_file(file):
```

Comment:
I would have only one method: `cleanup files`.  No need for a method with a single file as argument.

Reply:
Yeah I just figured having one for a single file would be nicer in the long run. Rather than having to put a single file into an array just to remove it with the other function.